package.json

```diff
@@ -8,7 +8,8 @@
   "repository": "https://github.com/paritytech/substrate-light-ui.git",
   "workspaces": {
     "packages": [
-      "packages/*"
+      "packages/*",
+      "node_modules/@substrate/ui-components"
```

Comment:
does this work?

Reply:
ah forgot to remove. It fixes one problem and reveals another :(